libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -179,6 +181,65 @@ bool IRGeneratorForStatements::visit(FunctionCall const& _functionCall)
 	return false;
 }
 
+bool IRGeneratorForStatements::visit(InlineAssembly const& _inlineAsm)
+{
+	auto const& loc = _inlineAsm.location();
+
+	solAssert(loc.start != 1 && loc.end != -1, "Expected source location");
+
+	vector<string> params, retParams, args;
+	// Prepare parameters and arguments
+	for (auto const& pair: _inlineAsm.annotation().externalReferences)
+	{
+		auto varDecl = dynamic_cast<VariableDeclaration const*>(pair.second.declaration);
+		solUnimplementedAssert(!!varDecl, "");
```

Comment:
You should probably also assert that it doesn't refer to a reference type variable for now - i.e. assert that ``isOffset`` and ``isSlot`` in the ``externalReferences`` are both false.